src/common/paths/legacy_query.yml

```diff
@@ -0,0 +1,114 @@
+get:
```

Comment:
Do you mean for this to be included in the contracts? If so,
you need to set the PATH in https://github.com/influxdata/openapi/blob/master/src/common/_paths.yml, regenerate the contracts, and push them up.

Reply:
Will the path be `/legacy/query`, similar to https://github.com/influxdata/openapi/blob/a9df1ff92683aedf316bdb2650e146e2bdc7e6c3/src/oss.yml#L49?

Reply:
I'm going to PR some suggestions to your branch and then I'll update based on your replies.